src/coreclr/src/jit/optcse.cpp

```diff
@@ -1664,7 +1664,7 @@ class CSE_Heuristic
                 largeFrame = true;
                 break; // early out,  we don't need to keep increasing frameSize
             }
-#elif _TARGET_ARM32
+#elif TARGET_ARM
```

Comment:
This should be `#elif defined(TARGET_ARM)` (similar change below for ARM64) -- not related to this change